bam_masterdata/cli/cli.py

```diff
@@ -727,7 +728,14 @@ def run_parser(
     required=True,
     help="OpenBIS collection name",
 )
-def parser(files_parser, project_name, collection_name):
+@click.option(
+    "--space_name",
+    "space_name-name",  # alias
+    type=str,
+    required=True,
```

Comment:
if the `space_name` defaults to openbis.username, I would delete this required option